package.json

```diff
@@ -37,16 +37,23 @@
   },
   "dependencies": {
     "@types/bitcore-lib": "0.15.1",
+    "@types/bs58": "3.0.30",
+    "@types/cuid": "1.3.0",
+    "@types/joi": "13.4.4",
     "@types/node-fetch": "1.6.9",
     "babel-cli": "6.26.0",
     "babel-plugin-module-resolver": "3.1.1",
     "bitcore-lib": "0.15.0",
+    "cuid": "2.1.4",
```

Comment:
You can remove bitcore-lib from the dependencies now that the signing is being replaced by jsonld-signatures.

Reply:
👍

Reply:
Done